app/src/org/odk/collect/android/tasks/FormLoaderTask.java

```diff
@@ -133,7 +136,7 @@ protected FECWrapper doInBackground(Uri... form) {
                 fis = new FileInputStream(formXml);
                 XFormParser.registerHandler("intent", new IntentExtensionParser());
                 XFormParser.registerStructuredAction("pollsensor", new PollSensorExtensionParser());
-                fd = XFormUtils.getFormFromInputStream(fis);
+                fd = XFormUtils.getFormFromInputStream(fis, new ImageRestrictionExtensionParser());
```

Comment:
Did this explicitly actually, after talking to @ctsims  about it. Basically the static pattern is a really terrible one, and actually causes unwanted behavior for this feature (having 2 extension parsers registered). I really should have refactored the others that are currently static to be non-static as well, but decided to leave it for another day separately.